README.md

```diff
@@ -1183,7 +1183,11 @@ class MyWidgetWithTsx extends Themed(WidgetBase)<MyProperties> {
 Widgets can be turned into [Custom Elements](https://www.w3.org/TR/2016/WD-custom-elements-20161013/) with
 minimal extra effort.
 
-Just create a `CustomElementDescriptor` factory and use the `@dojo/cli` build tooling to do the rest of the work,
+There are two ways to turn a widget into a custom element.
```

Comment:
can we get rid of the description of using the CustomElementDescriptor completely? we plan to deprecate it as user facing feature in the future.

Reply:
The README is updated now to only mention the decorator.